### DISCUSSION TITLE
RAD Being Strange

### DISCUSSION THREAD

**Discussion by SensoryTrash** [frntc/RAD#21] [Q&A]
Ok i have built a RAD and seem to be having some issues with it...
i checked all solder points etc.

here is my chips and source,,,

74HCT30	- 74HCT30D-Q100J Digikey
74CBTD3861 - SN74CBTD3861DWR  Sourced on Ebay none at Digikey
74LVC245 - 74LVC245AD,118	 Digikey
74LVC257 - 74LVC257ADB,112 Digikey
74LVC573 - 74LVC573AD,118 Digikey

the cartridge starts in the c64 screen as normal with 16384k rev at the bottom of screen
pressing the menu button locks up the pi...

pressing the reset button resets but only if you haven't pressed the menu button
The pi is powered by its own dedicated power supply and the SD image was thankfully provided by the creator and awesome chap... Thomas

**Comment by SensoryTrash** [comment]
![2023-06-22 09 06 56](https://github.com/frntc/RAD/assets/95701281/eab7ac83-0342-437b-a144-cab975650e63)

**Comment by frntc** [comment]
When the C64 hangs, can you measure the voltage levels for DMA (pin 20@CBTD3861), RESET (pin 21), RW (19), IRQ (17)? (you also find these signals on the expansion port and at various places on the C64-board)

**Comment by SensoryTrash** [comment]
Thank you got it all figured out i had some bad legs on a ship all fixed and loving it.
